cmd/acrp-rest/static/emp_dept/accountlinked.html

```diff
@@ -66,6 +66,11 @@
                         <a href="/" class="text-white bold hover:bg-gray-700 hover:text-white px-3 py-6 rounded-md text-lg">Travel</a>
                     </div>
                 </div>
+                <div class="flex-1 flex items-right justify-end sm:items-stretch sm:justify-end">
```

Comment:
Don't we need to update folder names `cmd/acrp-rest/static/<mode>/accountlinked.html`  based on changes https://github.com/trustbloc/sandbox/pull/813/files#diff-7473366dd614b41c1fa9b02dd689bfa5db78bc4d5c6e8d73250294d91ce79f8aR94 ?

Reply:
yes i wiil do in next pr. Will be done soon before morning

Reply:
Its gonna break the master, right ?

Reply:
its fixed now